src/ansys/fluent/core/launcher/launcher.py

```diff
@@ -52,7 +52,12 @@ def set_fluent_path(fluent_exe_path: Union[str, Path]) -> None:
 
     This supersedes the fluent path set in the environment variable
     """
-    FLUENT_EXE_PATH.append(str(fluent_exe_path))
+    if str(fluent_exe_path).endswith("fluent.exe"):
```

Comment:
@prmukherj or instead of creating a `str`, create a Path, access its `name` and do a more precise comparison.

Reply:
Added a Path.exists() check which should take care of it along with the sting check. Should be fine? Or should we do something else?

Reply:
Thank you for the suggestion. It's implemented and seems much better.